guizero/base.py

```diff
@@ -406,11 +406,20 @@ def _close_window(self):
     def update(self):
         self.tk.update()
 
+    def full_screen(self, keybind="<Escape>"):
+        """Make this window full screen and bind the Escape key (or given key) to exit full screen mode"""
+        self.tk.attributes("-fullscreen", True)
+        self.tk.bind(keybind, self.windowed)
+
+    def windowed(self, e):    # In the jungle, the mighty jungle...
+        """Change from full screen to windowed mode"""
+        self.tk.attributes("-fullscreen", False)
+
```

Comment:
We should clear the `<Escape>` event when full_screen is exited.

Using the event manager it would be something like:

```
self.events.remove_event("<FullScreen.Escape>")
```